lib/tenantmanager.js

```diff
@@ -276,23 +255,30 @@ exports = module.exports = {
     } else {
       logger.log('info', 'No filesystem required for tenant ' + tenant.name);
       callback(null);
-    }    
-    
+    }
+
   },
 
   /**
    * creates a tenant
    *
-   * @param {object} tenant - a fully defined tenant object
+   * @param {object} tenant - tenant details, only details on create tenant whitelist will be accepted
    * @param {function} callback - function of the form function (error, tenant)
    */
   createTenant: function (tenant, callback) {
     var self = this;
 
+    // Strip out fields that are not on whitelist
+    Object.keys(tenant).forEach(function(key) {
+      if (!CREATE_TENANT_WHITELIST.includes(key)) {
+        delete delta[key];
```

Comment:
getting ReferenceError: delta is not defined on install. Should this be `delete tenant[key]; `